src/components/Composer/implementation/index.native.tsx

```diff
@@ -57,6 +57,23 @@ function Composer(
         inputCallbackRef(autoFocus ? textInput.current : null);
     }, [autoFocus, inputCallbackRef, autoFocusInputRef]);
 
+    useEffect(() => {
+        if (!textInput.current || !textInput.current.setSelection || !selection || isComposerFullSize) {
+            return;
+        }
+
+        // We need the delay for setSelection to properly work for IOS.
```

Comment:
Mind putting a little more effort into this comment? We now have a great detailed explanation that should be easy to summarize. “We need the delay for setSelection to properly work for IOS” doesn’t really explain anything at all except that it's a platform workaround for some unknown issue.

Reply:
I'm sorry but we still don't know the correct root cause. can you suggest a comment?

Reply:
What about this?
```
// We need the delay for setSelection to properly work for IOS in bridgeless mode due to a react native internal bug of dispatching the event before the component is ready for it.
```

Reply:
Yeah is there some kind of issue related to the bug that you can link? It sounds like we might be able to clean this up in the future.

Reply:
I have linked to the explanation [here](https://github.com/Expensify/App/pull/50520#discussion_r1861960311) because all necessary info can be found from it 👍